README.md

```diff
@@ -41,6 +43,7 @@ Full configuration:
 cors / {
   origin            http://allowedSite.com
   origin            http://anotherSite.org https://anotherSite.org
+  origin_regex      anotherSite\.com$
```

Comment:
Can the example show off something you need the regex to be able to do? Perhaps `.+\.example\.com$`?